spec/migrations/migration_logs_to_elk_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+RSpec.describe 'migration ccdb logs to elk', isolation: :truncation, type: :migration do
+  let(:db) { Sequel::Model.db }
+  let(:tmp_migrations_dir) { Dir.mktmpdir }
+
+  before do
+    long_time_migration = <<-MIGRATION
+      Sequel.migration do
+        up do
+          10.times do
+            VCAP::Migration.logging('Still Migrating')
```

Comment:
When changing the method in `VCAP::Migration` as described above, this should be:
```
VCAP::Migration.logger.info(...)
```